homeassistant/components/vicare/sensor.py

```diff
@@ -815,6 +815,21 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         state_class=SensorStateClass.MEASUREMENT,
         value_getter=lambda api: api.getHumidity(),
     ),
+    ViCareSensorEntityDescription(
+        key="ventilation_demand",
+        translation_key="ventilation_demand",
+        value_getter=lambda api: api.getVentilationDemand().lower(),
+    ),
+    ViCareSensorEntityDescription(
+        key="ventilation_level",
+        translation_key="ventilation_level",
+        value_getter=lambda api: api.getVentilationLevel().lower(),
+    ),
+    ViCareSensorEntityDescription(
+        key="ventilation_reason",
+        translation_key="ventilation_reason",
+        value_getter=lambda api: api.getVentilationReason().lower(),
+    ),
```

Comment:
Let's have the possible options added in an `options` field and add the `ENUM` device class